integration/framework.go

```diff
@@ -457,3 +458,16 @@ func (f *framework) ApplyAWSConfigPatch(patch []PatchSpec, clusterName string) e
 
 	return nil
 }
+
+func (f *framework) GetVersionBundleVersion(bundle []versionbundle.Bundle, vType string) string {
+	var output string
+	for _, v := range bundle {
+		if (vType == "current" || vType == "") && !v.Deprecated && !v.WIP {
+			output = v.Version
+		}
+		if vType == "wip" && v.WIP {
+			output = v.Version
+		}
```

Comment:
What if there are multiple WIP or non-deprecated non-wip versions? It would be nice to have some guarantees.

Reply:
This looks definitely not that generic. What about having a circle template and just render it according to all version bundles available? I do not see a generic future direction with these conditionals.

Reply:
> What about having a circle template and just render it according to all version bundles available?

I'm afraid that wouldn't work, currently we need a static circleCI config file committed to the repo for the github integration to work. Looks like there's already a feature request for dynamic configuration https://discuss.circleci.com/t/dynamic-2-0-configuration/19287 once this is in place we can get rid of these hardcoded checks. Until then I was planning to update the e2e circleci config according to the existent version bundles but after we make sure that two concurrent builds work as expected.

Reply:
> Until then I was planning to update the e2e circleci config according to the existent version bundles but after we make sure that two concurrent builds work as expected.

For this we would just need to have enough jobs in the config (no need to set the actual version bundle versions in the CI config, but making sure that there enough for the current bundle), the dynamic retrieval would pick up the version bundle version for the job index.